tests/kernel/bigquery_tests.py

```diff
@@ -78,6 +129,14 @@ def test_query_cell(self, mock_default_context, mock_notebook_environment):
 
     # test query creation
     q1_body = 'SELECT * FROM test_table'
+
+    # no query specified
+    with self.assertRaises(Exception):
```

Comment:
Thanks for changing the other calls to assertRaisesRegexp. Maybe change this one also?